### ISSUE TITLE
Failed aws-cli installation : Illegal instruction (core dumped) from aws-crt-python

### ISSUE THREAD

**Issue by Dennis-Johnson** [awslabs/aws-crt-python#485]
### Describe the bug

Hi, trying to install the AWS CLI on Linux ARM64 resulted in a failure. This is related to [aws-cli #8008](https://github.com/aws/aws-cli/issues/8008), which suggested to file an issue here as well. 

After following the [instructions](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) for installation on Linux ARM, I get the following error on trying to invoke the CLI tool.
```
$ aws --version
Illegal instruction (core dumped)
```

### Expected Behavior

Expected to get the installed CLI version.

### Current Behavior

Broken installation of aws-cli from source bundle, possibly due to an issue in underlying aws-crt-python

### Reproduction Steps

```bash
$ aws --version
Illegal instruction (core dumped)
```

### System Info
```bash
$ uname -r
Linux agx 4.9.253-tegra #1 SMP PREEMPT Mon Jul 26 12:19:28 PDT 2021 aarch64 aarch64 aarch64 GNU/Linux
```

### GDB Output

```gdb
$  gdb /usr/local/aws-cli/v2/current/dist/aws

GNU gdb (Ubuntu 8.1.1-0ubuntu1) 8.1.1

Reading symbols from /usr/local/aws-cli/v2/current/dist/aws...(no debugging symbols found)...done.
(gdb) run
Starting program: /usr/local/aws-cli/v2/dist/aws
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/aarch64-linux-gnu/libthread_db.so.1".
process 11080 is executing new program: /usr/local/aws-cli/v2/dist/aws
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/aarch64-linux-gnu/libthread_db.so.1".

Program received signal SIGILL, Illegal instruction.
OPENSSL_cpuid_setup () at /aws-crt-python/crt/aws-lc/crypto/fipsmodule/cpucap/cpu_aarch64_linux.c:132
132	/aws-crt-python/crt/aws-lc/crypto/fipsmodule/cpucap/cpu_aarch64_linux.c: No such file or directory.
```

### aws-crt-python version used

Latest as of 28 June '23

### Python version used

3.10

### Operating System and version

Linux-4-Tegra (Ubuntu 18.04)

**Comment by yasminetalby** [comment]
Hello @Dennis-Johnson ,

Thank you very much for your submission. I have reached out to the aws-lc team internally to investigate the issue. I will post any updates or questions they might have here. 

Thank you very much for your time and collaboration.

Sincerely,

Yasmine

**Comment by andrewhop** [comment]
What CPU are you attempting to install the CLI on, can you run `lscpu` and attach the output?

**Comment by andrewhop** [comment]
Based on the line number `cpu_aarch64_linux.c:132` I believe you are hitting the same MIDR_EL1 issue that Valgrind was hitting. This was fixed in https://github.com/aws/aws-lc/pull/1006 which was released in [v1.10.0](https://github.com/aws/aws-lc/releases/tag/v1.10.0) of AWS-LC. The most recent version of the [Python CRT](https://github.com/awslabs/aws-crt-python/tree/fc7074dc36eb634ccedac5cb0c92a93af71c7d41/crt) includes [aws-lc @ 80c3943](https://github.com/awslabs/aws-lc/tree/80c394324382c78d2cbd783babaad1f612c1642d) which is [v1.11.0](https://github.com/aws/aws-lc/releases/tag/v1.11.0) which includes the fix. Can you try updating to a newer version of the AWS CLI?

**Comment by jmklix** [comment]
@Dennis-Johnson has updating to the latest version fixed this for you?
